client/src/components/profile/ProfileDetailCard.js

```diff
@@ -20,21 +19,33 @@ const useStyles = makeStyles(() => ({
   },
 }));
 
-function ProfileDetailCard() {
+function ProfileDetailCard({ profileDetails }) {
   const classes = useStyles();
-
   return (
     <Box className={classes.centerContent}>
       {/* Header Background Image */}
       <CardMedia
-        image={Background}
+        image={
+          profileDetails.headerPicture
+            ? profileDetails.headerPicture
+            : Background
+        }
         alt="bg"
-        style={{ width: "100%", height: "300px", objectFit: "cover" }}
+        style={{
```

Comment:
Are these inline styles resolved in dev already?

Reply:
They were but for some reason they no longer are. I have fixed it now.